client/packages/requisitions/src/ResponseRequisition/DetailView/AppBarButtons/CreateShipmentButton.tsx

```diff
@@ -16,6 +16,8 @@ export const CreateShipmentButtonComponent = () => {
   const t = useTranslation('distribution');
   const { mutate: createOutbound } = useResponse.utils.createOutbound();
   const isDisabled = useResponse.utils.isDisabled();
+  const isDisabledForAuthorisation =
```

Comment:
is there a reason for not just updating `useIsResponseDisabled`?
Currently, you can enter text into the reference and comment fields, but the update doesn't save, though there's no error given

Reply:
So I could show the statuses as 'disabled' 🤔  

Would it be confusing to not show the statuses and then have them reappear if approved?

Reply:
oh. yes, you've got me there. _Apart_ from that though.. it would be easier to disable using the current hook. 
What about just changing the response req `StatussChangeButton` then?

```
  if (isDisabled && !isDisabledForAuthorisation) return null;

  return (
    <SplitButton
      options={options}
      isDisabled={isDisabled}
etc
```

since that is the special case, all other disables are following the standard pattern

Reply:
[56be913](https://github.com/openmsupply/open-msupply/pull/1634/commits/56be9139fb981529302983fd13f84d305c5a02d3)